packages/clay-badge/README.md

```diff
@@ -0,0 +1,26 @@
+# clay-badge
+
+Clay Badge
+
+## Setup
+
+1. Install NodeJS >= v0.12.0 and NPM >= v3.0.0, if you don't have it yet. You
+can find it [here](https://nodejs.org).
+
+2. Install local dependencies:
+
+  ```
+  npm install
+  ```
+
+3. Build the code:
+
+  ```
+  npm run build
+  ```
+
+4. Open the demo at demos/index.html on your browser.
+
+## Contribute
+
+We'd love to get contributions from you! Please, check our [Contributing Guidelines](CONTRIBUTING.md) to see how you can help us improve.
```

Comment:
Hey @blzaugg, API's will show up in the claycss.com site. Does it make sense to have them in here as well? If so, we need to find a way to auto-generate them so it is easier to maintain.